dev/gqltest/search_test.go

```diff
@@ -732,7 +732,7 @@ func testSearchClient(t *testing.T, client searchClient) {
 				zeroResult: true,
 				wantAlert: &gqltestutil.SearchAlert{
 					Title:       "No results",
-					Description: "It looks like you may have meant to run a structural search, but it is not toggled.",
+					Description: "It looks like you may have meant to run a structural search, but the patterntype is not specified or it is not toggled.",
```

Comment:
```suggestion
					Description: "It looks like you're trying to run a structural search, but it is not enabled using the patterntype keyword or UI toggle.",
```

Copied the same suggestion to the other lines 😄